Automattic/jetpack
# VideoPress: Fix image URLs in the block editor

Fixes #28779

## Proposed changes:

* Adds a helper function that uses the `plugin_dir_url` function to determine the right URL for images added in the block editor.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






On simple site, where the problem is happening:
* Apply this branch to your simple site sandbox
* In the block inserter, click "Browse all"
* Search for VideoPress
* Hover over the VideoPress icon
* Check that the example image loads
* Add a VideoPress block and upload a video
* Select the block and go to the Color panel
* Check that the image loads correctly
* Copy the image URL
* Reset your sandbox back to production and disconnect from it
* Hard reload
* Go to the Color panel again and find the broken image
* Manually change its `src` to the previously copied value
* Check that the image loads correctly
* Check the images on JN and WoA as well, they should keep loading normally

Before | After
-|-
![2023-02-08_17-18-42](https://user-images.githubusercontent.com/8486249/217641890-4fc47a7c-cd1a-4358-9750-0f61853dff1b.png) | ![2023-02-08_17-20-06](https://user-images.githubusercontent.com/8486249/217643009-52f9c5e0-fe86-42a4-9775-3d5bfbe3f6eb.png)


Before | After
-|-
![2023-02-07_17-07-05](https://user-images.githubusercontent.com/8486249/217641752-22aa178a-d681-48dc-845a-996f6bc0d086.png) | ![2023-02-08_17-23-58](https://user-images.githubusercontent.com/8486249/217643044-995d40f2-dce0-440b-8964-637dc5be7af3.png)
